_docs/community.md

```diff
@@ -120,7 +124,120 @@ yaxis:
     min: 0
     max: 10
     opposite: true
+```
 
 which will produce something like the following...
 
 ![chart example](./assets/apex-chart-power-vs-cost.png)
+
+## Import and Export Rates Chart
+
+If you're looking to combine import and export rates then create a card with the configuration:
+
+
+```yaml
+type: custom:config-template-card
+entities:
+  - sensor.octopus_energy_electricity_21l4726127_2000021792976_current_rate
```

Comment:
you might want to remove the identifiers for your meters?

Reply:
Done